deploy/dashboards/generic-go.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: "monitoring.kiali.io/v1alpha1"
+kind: MonitoringDashboard
+metadata:
+  name: generic-go
```

Comment:
to fix the conflict this should be renamed "go", as well as the file name

Reply:
yeah, I just did that as part of the rebase